firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+TabToolbarDelegate.swift

```diff
@@ -235,11 +235,63 @@ extension BrowserViewController: TabToolbarDelegate, PhotonActionSheetProtocol {
         return [newTab, closeTab]
     }
 
+    func getTabToolbarRefactorLongPressActions() -> [[PhotonRowActions]] {
+        let newTab = SingleActionViewModel(title: .KeyboardShortcuts.NewTab,
```

Comment:
nit: these seem very similar to what we are using for `getMoreTabToolbarLongPressActions` and `getNewTabLongPressActions` could we consolidate. Not only to remove redundant code, but if there is a change to one, then it'll be consistent across all areas. What do you think?

Reply:
I had considered this, but ultimately scrapped the idea since it would result in a pretty long function. I went ahead and implemented it though, and don't think it's _too_ bad. Let me know what you think.

Reply:
Oh okay, sorry I wasn't clear. I think the individual methods are good, but we can refactor out the SingleActionViewModel. For example, there are two newPrivateTab constants, but maybe we can do `getNewPrivateTabAction`. The only distinction I saw at first glance is the iconString (which I would also question), so maybe we can do something like this?

```
func getNewPrivateTabAction(with iconString: String) -> PhotonRowActions {
 return SingleActionViewModel(title: .KeyboardShortcuts.NewPrivateTab,
                                                      iconString: iconString,
                                                      iconType: .Image) { _ in
                let shouldFocusLocationField = self.newTabSettings == .blankPage
                self.overlayManager.openNewTab(url: nil, newTabSettings: self.newTabSettings)
                self.openBlankNewTab(focusLocationField: shouldFocusLocationField, isPrivate: true)
                TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .newPrivateTab, value: .tabTray)
            }.items
}

Within each method:
let newPrivateTab = getNewPrivateTabAction()
```

Reply:
This is a nit, so I leave it up to you. However, I definitely prefer the previous implementation.

Reply:
Ahhh, totally misunderstood. I see now. Pushed [this commit](https://github.com/mozilla-mobile/firefox-ios/pull/21287/commits/228cc81ddf1d73f04079fc516d6596d071dded9c) with the changes. Just noticed you suggested added `iconString` as an arg. I used the refactor flags instead, so let me know what you think!